plugins/source/gcp/resources/services/compute/zones_test.go

```diff
@@ -18,6 +18,7 @@ func createZones(mux *httprouter.Router) error {
 	}
 	emptyStr := ""
 	zones.NextPageToken = &emptyStr
+	zones.Items = zones.Items[:1] // leave only 1
```

Comment:
What's the reason this is needed? I thought  `faker.FakeObject` only adds a single item per array

Reply:
just to ensure that we indeed only have 1 call in the children resolvers